src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -611,22 +609,6 @@ export default {
 			}
 		}
 
-		const activityFeedExpData = trackExperimentVersion(
-			this.apollo,
-			this.$kvTrackEvent,
-			'borrower-profile',
-			ACTIVITY_FEED_EXP,
-			'EXP-ACK-1037-MAR2024',
-		);
-		if (activityFeedExpData?.version === 'b') {
-			const response = await this.apollo.query({
-				query: loanActivitiesQuery,
-				variables: { loanId: this.loanId }
-			});
-
-			this.activities = response?.data ?? null;
```

Comment:
Looks like this data prop needs to be removed as well?

Reply:
It can also be removed from `data`

Reply:
thanks